webapp/client/src/lib/translations.js

```diff
@@ -87,12 +90,35 @@ const translations = {
   },
   lotseFlow: {
     taxNumber: {
+      taxNumberExists: {
+        labelText_one: "Haben Sie bereits eine Steuernummer?",
+        labelText_other: "Haben Sie bereits eine gemeinsame Steuernummer?",
```

Comment:
We're in JavaScript-land, shouldn't this be camelCased? Or nested:
```
        labelText: {
            one: "Haben Sie bereits eine Steuernummer?",
            other: "Haben Sie bereits eine gemeinsame Steuernummer?",
        },
```

Reply:
In fact, this is how it's done in i18nnext for plurals (with underscore): https://www.i18next.com/translation-function/plurals

Reply:
Oooh! Thanks! Looks a bit stupid, but I understand why they do it that way…